UnityProjectDP/Assets/Scripts/Visualization/ClassDiagram/Editors/ParsedEditor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using UnityEngine;
+
+namespace AnimArch.Visualization.Diagrams
+{
+    public static class ParsedEditor
+    {
+        public static Class UpdateNodeGeometry(Class newClass, GameObject classGo)
+        {
+            if (newClass.Left != 0)
+                return newClass;
+            var position = classGo.transform.localPosition;
+            newClass.Left = position.x / 2.5f;
+            newClass.Top = position.y / 2.5f;
+            return newClass;
+        }
+
+        public static void ReverseNodesGeometry()
+        {
+            foreach (var parsedClass in DiagramPool.Instance.ClassDiagram.GetClassList()) parsedClass.Top *= -1;
+        }
+
+        public static void AddAttribute(ClassInDiagram classInDiagram, Attribute attribute)
+        {
+            classInDiagram.ParsedClass.Attributes.Add(attribute);
+        }
+
+        public static void UpdateAttribute(ClassInDiagram classInDiagram, string oldAttribute, Attribute newAttribute)
+        {
+            var index = classInDiagram.ParsedClass.Attributes.FindIndex(x => x.Name == oldAttribute);
```

Comment:
Ak dobre pozeram, FindIndex ak sa nepodari najst vracia -1

Reply:
Navrhuješ sa vraciať z funkcie? Lebo akonáhle nenájde tak musí byť chyba niekde inde, napr pri načítavaní or smth a lepšie je dostať error na index nech ťa to upozorní podľa mňa. Lebo by k tomu malo dochádzať iba ak niečo nefunguje

Reply:
Tak, mozme tu aj nieco thrownut, nie?